includes/hooks.php

```diff
@@ -454,14 +454,12 @@ public static function publish_comment( $comment ) {
 			'last_updated' => current_time( 'mysql' ),
 		) );
 
-
 		// Log to activity table.
 		ap_activity_add( array(
-			'q_id'    => 'answer' === $post->post_type ? $post->post_parent: $post->ID,
-			'action'  => 'new_c',
-			'a_id'    => 'answer' === $post->post_type ? $post->ID: 0,
-			'c_id'    => $comment->comment_ID,
-			'user_id' => $comment->user_id,
+			'q_id'   => 'answer' === $post->post_type ? $post->post_parent: $post->ID,
+			'action' => 'new_c',
+			'a_id'   => 'answer' === $post->post_type ? $post->ID: 0,
+			'c_id'   => $comment->comment_ID,
```

Comment:
Please share more detail why you think `user_id` argument is not needed. Thanks

Reply:
This is also not our update..